.github/workflows/publish_python_docker.yml

```diff
@@ -94,8 +93,8 @@ jobs:
           scan: true
           image_extra_aliases: |
             ${{ env.IMAGE_NAME }}:${{ needs.calculate_version.outputs.next_version }}
-            ${{ github.ref == 'refs/heads/development' && format('{0}:{1}', env.IMAGE_NAME, 'latest') || '' }}
-            ${{ github.ref == 'refs/heads/development' && format('{0}/{1}:{2}', 'ghcr.io', env.IMAGE_NAME, 'latest') || ''}}
+            ${{ startsWith(github.ref, 'refs/heads/release-') && format('{0}:{1}', env.IMAGE_NAME, 'latest') || '' }}
+            ${{ startsWith(github.ref, 'refs/heads/release-') && format('{0}/{1}:{2}', 'ghcr.io', env.IMAGE_NAME, 'latest') || ''}}
```

Comment:
this will publish `latest` tag from any release branch.

e.g. you only have `release-0.1` with latest release `0.1.4`
* you create `release-0.2`  -> your `latest` tag applies to `0.2.0`
* someone reported a bug that could only be reproduced on `release-0.1`
* you applied the fix, and published to `release-0.1` branch
* -> new release `0.1.5` is being issued
* -> the tag `latest` applies to `0.1.5` while we still have `0.2.0`